pages/members/Member.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import styles from "./members.module.css"
+import Image from "next/image"
+import bannerImgDefault from "../../public/blank_banner.png"
+
+export default function Member(props) {
+ 
+  const profile_image = props.profile_image_url.replace("_normal", "")
+  const description = props.description.replace(/\n/g, ' ').trim()
+
+  const [profileLoad, setProfileLoad] = React.useState(false)
+
+  function showProfile() {
+    setProfileLoad(true)
+  }
+  
+  return (
+      <div className={styles.holdBox}>
+        <a href={`https://twitter.com/${props.screen_name}`} target="_blank" className={styles.member} rel="noopener noreferrer">
+
+        {props.profile_banner_url ? //checks if member has banner set. renders default if not
```

Comment:
use if else to swap the image source instead of redeclaring

Reply:
am using a ternary operator here to check if the user has a twitter banner and if not then render a default one.
`banner ? banner : default banner`. Seems clean to me, if not cleaner than if/else. 

The other ternary for profile_image is used to display a default image until the actual image is loaded and switches the CSS display property `none` from one to the other

Reply:
ok. I see an issue when soft refreshing the page that the default image is rendered but not the fetched one as the function I've passed to `onLoad` sets to true but on refresh is already true so doesnt trigger the change

Reply:
have fixed with 
`<img src={!!profile_image ? profile_image : "../images/blank_profile.png" } className={styles.profileImg}`, which is what I think you were asking for

Reply:
hmm.. actually this code is to check if exists is pointless as it will always exist if the fetch was successful. If it wasn't then the whole card wouldnt load anyway. 
May as well have as  `<img src={profile_image} className={styles.profileImg} />`